### PR TITLE
134 cmd

### PR DESCRIPTION
**Pull request by shawnps**

### CODE REVIEW

**Comment by shawnps** [comment]
@hermanschaaf started to write the command here, it's not ready yet but feel free to comment

File: cmd/main.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	"log"
+
+	"github.com/gojp/goreportcard/check"
+	"github.com/gojp/goreportcard/handlers"
+)
+
+var allScores []handlers.Score
+
+func main() {
+	dir := "."
```
Comment (shawnps) [inline cmd/main.go]: not sure the best thing to do here, add a flag?

File: cmd/main.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+		log.Fatalf("no .go files found")
+	}
+
+	err = check.RenameFiles(skipped)
+	if err != nil {
+		log.Println("Could not remove files:", err)
+	}
+	defer check.RevertFiles(skipped)
+
+	checks := []check.Check{
```
Comment (shawnps) [inline cmd/main.go]: how much of this do we want to share between the command and the handler? might not be worth sharing a ton of the code b/c it's so small and i like being able to see all the logic in one place. opinions welcome

File: cmd/main.go [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+		totalWeight += s.Weight
+	}
+	total /= totalWeight
+
+	grade := handlers.PercentToGrade(total * 100)
+
+	for _, score := range allScores {
+		fmt.Printf("%s: %.2f%%\n", score.Name, score.Percentage*100)
+	}
+	fmt.Println("Grade:", grade)
```
Comment (shawnps) [inline cmd/main.go]: do we want to show the grade or just the scores?

File: cmd/main.go [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/gojp/goreportcard/check"
+	"github.com/gojp/goreportcard/handlers"
+)
+
+var allScores []handlers.Score
```
Comment (shawnps) [inline cmd/main.go]: i found it kind of weird calling this `handlers.Score`, do we want to add another package for the Grade and Score types maybe?

**Comment by hermanschaaf** [comment]
Since this is still WIP and hasn't been worked on in a while, will close for now. If anyone wants to pick it up, let us know.

**Pull request closed without merging**